Misc/NEWS.d/next/Windows/2022-09-07-00-11-33.gh-issue-96577.kV4K_1.rst

```diff
@@ -0,0 +1 @@
+Fixes a potential buffer overrun in :mod:`msilib`.
```

Comment:
You may move this to the security category, but I'm fine with the Windows category.